go.mod

```
@@ -1,6 +1,8 @@
 module github.com/grafana/xk6-redis
 
-go 1.20
+go 1.23.0
+
+toolchain go1.23.7
```

```suggestion
toolchain go1.23.10
```

This is the latest security update at this point , alhtough .11 is coming tomorrow, still better to bump it a bit.

---

Yeah, good point! I used the same as in k6 for coherence, because it defaulted to 1.24, which is the one I have installed locally. But yeah, of course it's better to use the latest patch. Thanks!